docs/versioned_docs/version-2.11.0/additional-info/performance.md

```diff
@@ -1,6 +1,11 @@
 ---
 id: performance-testing
 title: Performance testing of OpenEBS
+keywords: 
+ - Performance testing
+ - Testing
```

Comment:
Let's Please add proper keywords. Testing repeated 3 times in almost the same context.

Reply:
Have tried to add few more keywords and removed the repetitive keyword.